tests/backup/backup_restore_basic_test.go

```diff
@@ -895,3 +897,172 @@ var _ = Describe("{CustomResourceRestore}", func() {
+			backupInspectRequest := &api.BackupInspectRequest{
+				Name:  nextScheduleBackupName.(string),
+				Uid:   nextScheduleBackupUid,
+				OrgId: orgID,
+			}
+			resp, err := Inst().Backup.InspectBackup(ctx, backupInspectRequest)
+			dash.VerifyFatal(err, nil, fmt.Sprintf("Inspecting next schedule backup named [%s] and uid [%s]", nextScheduleBackupName, nextScheduleBackupUid))
+			backedUpNamespaces := resp.GetBackup().GetNamespaces()
+			log.InfoD("Namespaces in next schedule backup named [%s] and uid [%s] are [%v]", nextScheduleBackupName, nextScheduleBackupUid, backedUpNamespaces)
+			for _, namespace := range appNamespaces {
+				dash.VerifyFatal(strings.Contains(strings.Join(backedUpNamespaces, ","), namespace), true, fmt.Sprintf("Checking the new application namespace [%s] against the next scheduled backup named [%s]", namespace, nextScheduleBackupName))
+			}
+		})
+		Step("Restore new application namespaces from next schedule backup", func() {
+			log.InfoD("Restoring new application namespaces from next schedule backup")
+			ctx, err := backup.GetAdminCtxFromSecret()
+			dash.VerifyFatal(err, nil, "Fetching px-central-admin ctx")
+			namespaceMapping := make(map[string]string)
+			for _, namespace := range appNamespaces {
+				namespaceMapping[namespace] = namespace
+			}
+			log.InfoD("Namespace mapping used for restoring - %v", namespaceMapping)
+			restoreName = fmt.Sprintf("%s-%s", "test-restore", RandomString(4))
+			err = CreateRestore(restoreName, nextScheduleBackupName.(string), namespaceMapping, destinationClusterName, orgID, ctx, make(map[string]string))
+			dash.VerifyFatal(err, nil, fmt.Sprintf("Creating restore [%s]", restoreName))
+		})
+	})
+
+	JustAfterEach(func() {
+		defer EndPxBackupTorpedoTest(contexts)
+		ctx, err := backup.GetAdminCtxFromSecret()
+		dash.VerifySafely(err, nil, "Fetching px-central-admin ctx")
+		scheduleUid, err := GetScheduleUID(scheduleName, orgID, ctx)
+		dash.VerifySafely(err, nil, fmt.Sprintf("Fetching uid of schedule named [%s]", scheduleName))
+		allScheduleBackupNames, err := Inst().Backup.GetAllScheduleBackupNames(ctx, scheduleName, orgID)
+		dash.VerifySafely(err, nil, fmt.Sprintf("Fetching all schedule backup names of schedule named [%s]", scheduleName))
+		log.InfoD("Deleting schedule named [%s] along with its backups [%v] and schedule policies [%v]", scheduleName, allScheduleBackupNames, []string{periodicSchedulePolicyName})
+		err = DeleteSchedule(scheduleName, scheduleUid, periodicSchedulePolicyName, periodicSchedulePolicyUid, orgID)
+		dash.VerifySafely(err, nil, fmt.Sprintf("Verifying deletion of backup schedule named [%s]", scheduleName))
+		err = DeleteRestore(restoreName, orgID, ctx)
+		dash.VerifySafely(err, nil, fmt.Sprintf("Deleting restore [%s]", restoreName))
+		opts := make(map[string]bool)
+		opts[SkipClusterScopedObjects] = true
+		log.Info("Deleting deployed namespaces - %v", appNamespaces)
```

Comment:
Use `Infof`

Reply:
Good catch, Thanks! I will make the necessary changes.